python/tvm/relax/transform/lazy_transform_params.py

```diff
@@ -188,24 +199,24 @@ def visit_var_(self, var: relax.Var) -> None:
         return super().visit_var_(var)
 
     def visit_var_binding_(self, binding: relax.VarBinding) -> None:
-        if binding.var == self.out_tuple_var:
+        if self.fset_item is not None and binding.var == self.out_tuple_var:
```

Comment:
It looks like the handling of `set_item` and of `get_item` is largely independent.  Rather than having them as two features of a single mutator, which requires the `self.fset_item is not None` check to be repeated throughout, can we split it into two distinct mutators?  One mutator replaces access of parameters with `get_item`.  The other mutator replaces binding of output with `set_item`.  If both features are desired, then both mutators are applied.

Reply:
sounds good. let me try